RevenueCatUI/Templates/Template5View.swift

```diff
@@ -15,6 +15,8 @@ import RevenueCat
 import SwiftUI
 
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(tvOS, unavailable)
```

Comment:
Do we need to remove the `watchOS` from the lines above each of these?

Reply:
If you don't add it it assumes the requirement matches the deployment target in the target which is 6.2 so it fails to compile.